pygeofilter/backends/sqlalchemy/filters.py

```diff
@@ -7,6 +6,8 @@
 from pygeoif import shape
 from sqlalchemy import and_, func, not_, or_
 
+from ... import values
```

Comment:
What does this import do?  I don't see it referenced in the rest of the PR?

Reply:
I think I just forgot to remove that one after I implemented the proposed solution :sweat: - Removed it now